samples/nrf9160/http_application_update/src/main.c

```diff
@@ -89,7 +89,8 @@ static void app_dfu_transfer_start(struct k_work *unused)
 	retval = fota_download_start(CONFIG_DOWNLOAD_HOST,
 				     CONFIG_DOWNLOAD_FILE,
 				     sec_tag,
-				     apn);
+				     apn,
+				     0);
```

Comment:
I sugggest we add a define called something like  "USE_FRAGMENT_SIZE_DEFAULTS" or similar